README.md

```diff
@@ -13,39 +13,119 @@ Artifacts in support of ODH Edge use cases that show integration with Red Hat Ad
+1. Create the MultiClusterHub.
+   In OpenShift Console in Menu > Operators > Installed Operators > Advanced Cluster Management for Kubernetes >
+   the MultiClusterHub tab, use the "Create MultiClusterHub" button to start the installation.
+1. Refresh the Web OpenShift Console.
+1. Create a cluster set for your near edge clusters.
+   In OpenShift Console it can be done in All Clusters menu > Infrastructure > Clusters > Cluster sets tab.
+   This repository assumes name `poc-near-edge` in `clusterSets`
+   in [acm/registration/near-edge/base/near-edge.yaml](acm/registration/near-edge/base/near-edge.yaml)
+   so update that YAML file if you use different name.
+1. Import the near edge cluster(s).
+   In OpenShift Console it can be done in All Clusters menu > Infrastructure > Clusters > Cluster list tab.
+   In the Cluster set popup menu, select the cluster set you created for the near edge clusters.
+   If you use the Import mode of "Enter your server URL and API token for the existing cluster",
+   obtain the address of the server and the token on the other cluster's OpenShift Console 
+   at the top-right corner user menu > Copy login command, for an admin user.
+
+#### Create namespaces for applications
+
+For each of the deployed models-turned-into-application, we will create separate namespace/project on the ACM hub cluster.
+The ACM setup in this repository expects namespaces `tensorflow-housing-app` and `bike-rental-app`.
+
+As a regular user in OpenShift Console, in the local-cluster > Developer menu, Project popup, use the "Create Project" button;
+alternatively, run
+```bash
+oc new-project tensorflow-housing-app
+oc new-project bike-rental-app
+
+```
+
+#### Bind namespaces to the cluster set
+
+The newly created namespaces need to be bound the cluster set with the near edge clusters.
+
+On the ACM hub cluster, in OpenShift Console in the All Clusters menu > Infrastructure > Clusters > Cluster sets tab,
+click the three-dot button at the right of the cluster set, select Edit namespace bindings, and add the namespaces you've created.
+
+This is the last step that needs to be done as an admin user with `cluster-admins` privileges.
+
+#### Deploy the applications to remote clusters
+
+With the remote near edge clusters in the cluster set and the cluster set bound to the application namespaces,
+it is now possible to deploy the applications to remote clusters through
+ACM Channels, Subscriptions, Placements, and Applications.
+
+We will create the objects in the respective namespaces from the `acm/registration/` directory structure.
+
+However, it is important to note that the actual application configuration, stored in `acm/odh-edge/`,
+does not come from our local git repository checkout.
+Instead, [acm/registration/near-edge/overlays/tensorflow-housing-app/kustomization.yaml](acm/registration/near-edge/overlays/tensorflow-housing-app/kustomization.yamlacm/registration/near-edge/overlays/tensorflow-housing-app/kustomization.yaml)
```

Comment:
```suggestion
Instead, [`acm/registration/near-edge/overlays/tensorflow-housing-app/kustomization.yaml`](acm/registration/near-edge/overlays/tensorflow-housing-app/kustomization.yamlacm/registration/near-edge/overlays/tensorflow-housing-app/kustomization.yaml)
```

Reply:
Applied in 49aadf7f7022a32884be3d91d13200f348ad82ab.